nestor_api/api/api_routes/workflow/workflow.py

```diff
@@ -0,0 +1,60 @@
+"""Define the workflows routes."""
+from http import HTTPStatus
+import traceback
+
+from nestor_api.adapters.git.provider import get_git_provider
+import nestor_api.lib.config as config_lib
+import nestor_api.lib.workflow as workflow_lib
+from nestor_api.utils.logger import Logger
+
+
+def init_workflow(organization, app):
+    """Initialize workflow of an application."""
+    Logger.debug(
+        {"app": app, "organization": organization},
+        "[/api/workflow/init/:org/:app] Workflow initialization started",
+    )
+
+    report = None
+    try:
+        # Retrieve project configuration
+        project_config = config_lib.get_project_config()
+
+        git_provider = get_git_provider(project_config)
+        report_status, report = workflow_lib.init_workflow(organization, app, git_provider)
+
+        if report_status == "fail":
+            status = HTTPStatus.INTERNAL_SERVER_ERROR
+            message = "Workflow initialization failed"
+        elif report_status == "success":
+            status = HTTPStatus.OK
+            message = "Workflow initialization succeeded"
+        else:
+            raise ValueError(f"Unexpected status: '{report_status}'")
+
+        # HTTP Response
+        Logger.info(
+            {"organization": organization, "app": app, "report": report},
+            f"[/api/workflow/init/:org/:app] {message}",
+        )
+        return (
+            {"organization": organization, "app": app, "report": report, "message": message},
+            status,
+        )
+    # pylint: disable=broad-except
+    except Exception as err:
+        Logger.error(
+            {"organization": organization, "app": app, "report": report, "err": str(err)},
+            "[/api/workflow/init/:org/:app] Workflow initialization failed",
+        )
+        print(traceback.print_exc())
```

Comment:
The Logger should now handle the printing of the stacktrace since https://github.com/ChauffeurPrive/nestor-api/pull/56.
Of course, don't hesitate to double-check in case I mixed things up here 🙏 
```suggestion
        Logger.error(
            {"organization": organization, "app": app, "report": report},
            "[/api/workflow/init/:org/:app] Workflow initialization failed",
        )
```

Reply:
No you're right, it is something I used for debugging before, should have put a "TODO remove" comment ;)